packages/hiro-graph-client/index.d.ts

```diff
@@ -372,6 +373,11 @@ export interface TimeseriesResponse {
   value: string;
 }
 
+export interface TimeseriesValue {
+  timestamp: number;
+  value: PlainObject | number | string | Array<string | number | PlainObject[]>;
```

Comment:
I was wondering, why does the `Array<...>` contain `PlainObject[]`, not just `PlainObject`?

Reply:
oh, thanks! my mistake!
I will fix it!

Reply:
done!